packages/itwinui-react/src/core/Header/HeaderSplitButton.tsx

```diff
@@ -52,6 +52,7 @@ export const HeaderSplitButton = React.forwardRef((props, forwardedRef) => {
         onHide={React.useCallback(() => setIsMenuOpen(false), [])}
       >
         <ButtonBase
+          title='Dropdown menu button'
```

Comment:
please don't use `title`. https://www.tpgi.com/using-the-html-title-attribute-updated/

`aria-label` should work fine.

and "Dropdown menu button" is not good as the label; but at the same time we don't really know what will be inside the dropdown, so the user should be the one providing it.

i'd say lets go with just "More" for now. in the future, we can make it customizable (#1444)

```suggestion
          aria-label='More'
```

Reply:
Thanks for sharing the link. I didn't realize that `title` was that problematic.